File: examples/src/Text/EffectTextBoxParagraph.cs
import aspose.slides as slides
using Aspose.slides.Animation
using Aspose.slides.Examples.CSharp
using System
using System.Collections.Generic
using System.Linq
using System.text
using System.Threading.Tasks

namespace CSharp.text
{
	class EffectTextBoxParagraph
	{
		public static void Run()
		{
			#ExStart:EffectTextBoxParagraph
			dataDir = RunExamples.GetDataDir_Charts()
			using (Presentation pres = new Presentation(dataDir + "Test.pptx"))
			{
				ISequence sequence = pres.slides[0].timeline.main_sequence
				autoShape = (IAutoShape)pres.slides[0].shapes[1]

				foreach (IParagraph paragraph in autoShape.text_frame.Paragraphs)
				{
					IEffect[] effects = sequence.GetEffectsByParagraph(paragraph)

					if (effects.Length > 0)
						print("Paragraph \"" + paragraph.text + "\" has " + effects[0].type + " effect.")
				}
			}
			#ExEnd:EffectTextBoxParagraph
		}
		}
}

File: examples/src/Text/EffectTextBoxParagraph.py
import aspose.slides as slides


#ExStart:EffectTextBoxParagraph
dataDir = "./examples/data/"
outDir = "./examples/out/"

with slides.Presentation(dataDir + "text_add_animation_effect.pptx") as pres:
	sequence = pres.slides[0].timeline.main_sequence
	for autoShape in pres.slides[0].shapes:
		for paragraph in autoShape.text_frame.paragraphs:
			effects = sequence.get_effects_by_paragraph(paragraph)

			if len(effects) > 0:
				print("Paragraph \"" + paragraph.text + "\" has " + str(effects[0].type) + " effect.")
#ExEnd:EffectTextBoxParagraph
